src/components/Shared/ScatterPlot/ScatterPlotComponent.tsx

```diff
@@ -21,17 +24,27 @@ export class ScatterPlotComponent extends LinePlotComponent {
         }
     }
 
-    private getScatterColor(value: number, min: number, max: number, toColor: number): string {
+    private getScatterColor(value: number, min: number, max: number, toColor: number, outRange: boolean): string {
         let percentage = (value + Math.abs(min)) / (Math.abs(min) + Math.abs(max));
         let hue = (percentage * toColor).toString(10);
-        return ["hsl(", hue, ",100%,50%)"].join("");
+        if (outRange) {
+            return ["hsla(", hue, ",0%,50%,", this.opacityOutRange, ")"].join("");
```

Comment:
we should use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for things like this. Fixing.